code/modules/awaymissions/corpse.dm

```diff
@@ -12,6 +12,8 @@
 	var/instant = FALSE	//fires on New
 	var/flavour_text = "The mapper forgot to set this!"
 	var/faction = null
+	var/permanent = FALSE	//Set to true for infinite mob spawn I guess.
```

Comment:
You could make this a var/uses instead and have -1 be infinite in case someone wants to do that sometime